_test/struct46.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import "fmt"
+
+type A struct {
+	B string
+	C D
+}
+
+type D struct {
+	E *A
+}
+
+func main() {
+	a := &A{B: "b"}
+	a.C = D{E: a}
+	fmt.Println(a.C.E.B)
+}
+
```

Comment:
```suggestion

// Output:
// b

```

Reply:
Otherwise the test is skipped with message: `has no comment 'Output:' or 'Error:'` when `go test`.

Reply:
Yeah, forgot to put it back. Thanks